native/src/routes/Pois.tsx

```diff
@@ -249,12 +250,14 @@ const Pois = ({ pois: allPois, language, cityModel, route, navigation }: PoisPro
         setSheetSnapPointIndex={setSheetSnapPointIndex}
         featureCollection={embedInCollection(features)}
         selectedFeature={currentFeatureOnMap ?? null}
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        bottomSheetHeight={getBottomSheetSnapPoints(deviceHeight)[sheetSnapPointIndex]!}
         locationPermissionGranted={!!coordinates}
         onRequestLocationPermission={requestAndDetermineLocation}
         iconPosition={
-          sheetSnapPointIndex < BOTTOM_SHEET_SNAP_POINTS.length - 1
+          sheetSnapPointIndex < getBottomSheetSnapPoints(deviceHeight).length - 1
             ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-              BOTTOM_SHEET_SNAP_POINTS[sheetSnapPointIndex]!
+              getBottomSheetSnapPoints(deviceHeight)[sheetSnapPointIndex]!
```

Comment:
Please extract this to a separate const such that disabling eslint is only needed once.
Optional: snapPoints are also calculated 4 times atm and could be extracted.